clients/java/src/main/java/io/camunda/zeebe/client/api/search/FinalSearchQueryStep.java

```diff
@@ -21,5 +21,6 @@
 
 public interface FinalSearchQueryStep<T> extends FinalCommandStep<SearchQueryResponse<T>> {
```

Comment:
❓ At some point, we need to move this class to the new package name as well. Better we do that now, I guess? Can this be considered a breaking change? 🙈

Reply:
I think this is not a breaking change as should be a new feature

Reply:
Ah yeah, true, nice catch! 👍